### PR TITLE
Task 10

### PR DESCRIPTION
**Pull request by VarshaJacob**

### CODE REVIEW

**Comment by VarshaJacob** [comment]
comment_count not implemented in be; working on it

File: src/api.js [hunk 1]
```diff
@@ -10,12 +10,22 @@ export const getTopics = () => {
     })
 };
 
-export const getArticles = (topic) => {
+export const getArticles = (sort_by,order,topic) => {
```
Comment (RichardNeat) [inline src/api.js:13]: We could refactor this to be a lot tidier using axios options. Have a look at the docs to see how we can build params into the url query conditionally.

File: src/components/SortList.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import {Link} from 'react-router-dom';
+
+const SortList = () => {
+    const sortList =['date','votes']
+    const orderList=['asc','desc']
+
+    const sortSubmit = (sort) => {
+        console.log(sort)
```
Comment (RichardNeat) [inline src/components/SortList.jsx]: please remove any stray console logs

**Review by RichardNeat — CHANGES_REQUESTED** [review]
Hi Varsha - I currently can't sort by, for example, votes and ascending. The way you have it laid out the url updates with ?sort_by=votes then when I click ascending it updates to remove the sort_by option and replaces it with ?order=asc and is no longer sorting by votes. Could you have a look and see what you think? Cheers :)

**Pull request closed without merging**